vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -756,6 +793,26 @@ public boolean isSubscription() {
         return (isPossiblePurchase() && isPurchased(mMetadata.mInteractions.mSubscribe));
     }
 
+    /**
+     * Helper to determine if this video is part of the SVOD library and included in the subscription. If wanting
+     * to know if a user also has access to the video, use {@link #canAccessVideoInSvod()}
+     *
+     * @return true if this video can be accessed with an SVOD subscription
+     */
+    public boolean isSvod() {
+        return mMetadata.getInteractions().getSvod() != null;
+    }
+
+    /**
+     * Helper to determine if the video is part of the SVOD library and that the user currently has access to it.
+     * This will be true if the user has purchased an SVOD subscription that is active.
+     *
+     * @return true if this video can be accessed with an SVOD subscription and the user's subscription is active
+     */
+    public boolean canAccessVideoInSvod() {
```

Comment:
Since they are helpers around metadata and pertain to all videos, I think it is OK to stay in this class.